dist/docs/guide/customization/promote-customization.adoc

```diff
@@ -1,6 +1,33 @@
 ### Promoting custom artifacts
```

Comment:
When is this operation needed? If I have "customization bundle" (zip), I can apply it as such, without having custom repository?

But I won't be able to use rollback operation in future, unless I have the customization bundle artifacts copied to custom repo?

Reply:
Currently the customization bundle is just a mechanism for distributing changes. It's not directly consumed by prospero, it has to be extracted and the server needs to be subscribed to it. In a simple case the extracted content of the bundle basically become the customization repository.

The idea of promoting artifacts is about updating that repository when new version of customization bundle is available.